### ISSUE TITLE
Bulk upload of non-sequence images through web uploader should auto split

### ISSUE THREAD

**Issue by DaCor1** [mapillary/mapillary_issues#941]
There are a lot of people around who have geotagged images but when they get bulk uploaded through the web uploader they create weird sequence with long lines between images. 

In the web uploader, when being uploaded, it would be good if there was an option to tick a box to say "These 400 images are not a single sequence, so should eb treated as individual images" etc etc

Alternatively, instead of waiting for people to request splits at distances greater than 300 meters, it should be automatic where the distance is greater than 1000 meters in a perfectly straight line

**Comment by Skippern** [comment]
+1

And it should be possible to request splitting of existing sequences, both your own and others

**Comment by DaCor1** [comment]
That's already possible Ann, just use the edit function when viewing an
image

**Comment by Skippern** [comment]
Sorry, found the split sequence in the edit just after I posted…

Still, auto-split sequences would be a big +

Aun Johnsen

**Comment by jesolem** [comment]
Agree, although individual images are not so useful, we should apply distance splitting to upload.

**Comment by DaCor1** [comment]
I agree they are not so on their own, however, when you combine them with regular mapillary images they become useful

Especially if you think of all the people on holidays snapping photos of historic sites etc

**Comment by knikel** [comment]
This will be a feature in the next iterations of the uploader (web + desktop)

**Comment by DaCor1** [comment]
Excellent

When you say desktop, do you mean through the use of scrips or an actual desktop program?

**Comment by knikel** [comment]
@DaCor1: Actual desktop program 👍

**Comment by DaCor1** [comment]
Omg awesome!

**Comment by knikel** [comment]
Assigning @amahon to consider this as a feature.

**Comment by Kaldea** [comment]
_mapillary_issues_ is no longer an active repository.

Let’s continue discussing features in our forum! We will import this issue as topic to the [Mapillary Forum](http://forum.mapillary.com) in the coming weeks. We’ll notify here once it’s been done. See you there!
